### ISSUE TITLE
Enhancement: при вызове меню в полноэкранном режиме отображать строку статуса

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#21]
```
Если есть возможность, сделайте пожалуйста 
так, чтобы в полноэкранном режиме (когда 
строка статуса скрыта), при вызове меню она 
отображалась. Это позволит видеть время, 
заряд, получить доступ к уведомлениям, без 
закрытия/сворачивания приложения.

```

Original issue reported on code.google.com by `mindphaser2k6` on 17 Aug 2011 at 9:12

**Comment by GoogleCodeExporter** [comment]
Original comment by `mc.creat` on 17 Aug 2011 at 7:46
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Implemented both in Trunk and 1.1.x
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 18 Aug 2011 at 11:23
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Alexander.V.Kasatkin@gmail.com` on 20 Mar 2012 at 5:53
- Added labels: **fullscreen**
